DOC-3132: Dropping files on the placeholder would sometimes insert th… (#3598)
tinymce/tinymce-docs: modules/ROOT/pages/7.7.0-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Image Optimizer

The {productname} {release-version} release includes an accompanying release of the **Image Optimizer** premium plugin.

**Image Optimizer** includes the following fixes and improvements.

==== Dropping files on the placeholder would sometimes insert the image in an incorrect location.
// #TINY-11643

Previously, an issue was identified where the drag-and-drop functionality on an image placeholder in a table would insert the image outside of the table cell.
This resulted in unpredictable behavior when users dragged and dropped an image into the image placeholder.

> {productname} {release-version} resolves this issue by making it so that that the image placeholder handles the drop event instead of the editor content element. This ensures predictable and consistent drag-and-drop behavior on image placeholders in Uploadcare.

Reviewer: ```suggestion
{productname} {release-version} resolves this issue by making it so that that the image placeholder handles the drop event instead of the editor content element. This improvement provides predictable and consistent drag-and-drop behavior for image placeholders in **Image Optimizer**.
```
